PatientRegistrationApp/src/App.vue

```diff
@@ -331,7 +331,9 @@ export default {
 			this.goToPage(config.registrationPages.HOUSEHOLD_HOME_ADDRESS_PAGE);
 		},
 		verifyHouseholdContactInfo() {
-			this.goToPage(config.registrationPages.HOUSEHOLD_PERSONAL_INFO_1_PAGE);
+			this.$refs.householdcontactinfo.verifyFormContents() ?
+			this.goToPage(config.registrationPages.HOUSEHOLD_PERSONAL_INFO_1_PAGE) :
+			this.goToPage(config.registrationPages.HOUSEHOLD_CONTACT_INFO_PAGE);
```

Comment:
What does this variable name actually refer to -- `HOUSEHOLD_PERSONAL_INFO_1_PAGE`. At any rate, it should be renamed to be more clear and specific for what it is.

Reply:
Upon reviewing the config file, is `HOUSEHOLD_REGISTER_NUMBER_PAGE` referencing phone number? That should be explicit then...

Reply:
HOUSEHOLD_PERSONAL_INFO_1_PAGE is the identifier for the HouseholdPersonalInfo_1.vue page.
HOUSEHOLD_REGISTER_NUMBER_PAGE is the identifier for the HouseholdRegisterNumber.vue page.

If/when we create design specifications, we could clearly associate the vue component names (and enumerations) to the layout of the page so that there is no confusion.

Reply:
I'm still not following the `_1.vue` page syntax. Do you mean that this is the personal info for the first person? I think the name we're using could be more descriptive.

Reply:
I changed the enumeration to HOUSEHOLD_PERSONAL_INFO_PATIENT_1_PAGE; this is the page that allows you to enter the personal information of patient #1.

Reply:
More clear, thanks